radeon-profile/gpu.cpp

```diff
@@ -265,12 +265,24 @@ void gpu::setForcePowerLevel(const QString &newForcePowerLevel) {
 }
 
 void gpu::setPwmValue(unsigned int value) {
+    // If the PC is sent to sleep (or hibernate) it can happen that PWM is
+    // disabled (by the OS..?) and that we have to re-enable it, if needed,
+    // after returning from sleep
+    static QFile pwmEnableFile(getDriverFiles().hwmonAttributes.pwm1_enable);
```

Comment:
It can be without static I think

Reply:
It could definitely be non-static. But I thought since the file is always the same it would suffice to construct the QFile object once and re-use it after that to gain a bit of performance (because this function is called quite frequently if I'm correct).

But if you prefer to have it non-static it's no problem for me.